agent/consul/leader_test.go

```diff
@@ -956,6 +956,7 @@ func TestLeader_ChangeServerID(t *testing.T) {
 	})
 }
 
+//Sometimes passed Sometimes failed
```

Comment:
```suggestion
```

I am sorry if this test is flaky, this is something we are actively working on. But such a comment doesn't help, which is why I think it shouldn't be there.

Reply:
I would got health check definition where get service from /health/service/:service or /health/checks/:service